pkg/destroy/bootstrap/bootstrap.go

```diff
@@ -31,6 +31,20 @@ func Destroy(dir string) (err error) {
 	}
 
 	if platform == openstack.Name {
+		if path := os.Getenv("OS_CLIENT_CONFIG_FILE"); path == "" {
```

Comment:
This duplicates what we have in [PreTerraform()](https://github.com/openshift/installer/blob/e6e2400/pkg/asset/cluster/openstack/openstack.go#L19)... Should this be wrapped in a function?

Reply:
Done. I ended up using that same function and removing the parameters that were not necessary.